lib/babel-istanbul-instrumenter.js

```diff
@@ -0,0 +1,256 @@
+/**
```

Comment:
If we are going to check this file into the repo in ES5, and isparta is unmaintained we should probably just rewrite this in idiomatic ES5 style instead of using the babel transpiled version of the isparta code.

Specifically, things like the following make life a little harder:

```js
var _this = _possibleConstructorReturn(this, Object.getPrototypeOf(Instrumenter).call(this));
//...snip...
var result = this._r = (0, babelTransform)(code, extend({}, this.babelOptions, { filename: fileName }));
```